Remove directional language: above pt.2 (#3426)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/new_features.md

In the following scenario, let's assume that you first upgraded to the Braze SDKs which supports this filter in version `2.0.0` of your app.

> Once Braze receives data from version 2.0.0 of your app, you can target users less than or greater than this version.

Author: ```suggestion
Once Braze receives data from version 2.0.0 of your app, you can target users with earlier or later versions.
```
